.pre-commit-config.yaml

```diff
@@ -7,5 +7,7 @@ repos:
       - id: helm-lint
       - id: kubepug-minimum
       - id: kubepug-recommended
-      # We will re-enable this via OPSEXP-703
-      # - id: checkov-helm
\ No newline at end of file
+  - repo: https://github.com/Alfresco/acs-deployment
+    rev: 5c9347de2a3ec7e3281f1b5e8da71e395f79c2a6
```

Comment:
Are there any implications to using a specific revision here?

Reply:
a revision is mandatory, however it cannot be the master branch. I know it looks ugly, didnt find a way around it for now, however on the next release of the chart we will be able to use the tag.